pkg/models/plan_collaborator.go

```diff
@@ -40,3 +41,19 @@ const (
 	TeamRoleOACT       TeamRole = "OACT"
 	TeamRolePayment    TeamRole = "PAYMENT"
 )
+
+func ConvertStringArrayToTeamRoles(sa pq.StringArray) []TeamRole {
+	roles := make([]TeamRole, len(sa))
+	for i, s := range sa {
+		roles[i] = TeamRole(s)
+	}
+	return roles
+}
```

Comment:
We already actually have a generic function for this!

https://github.com/CMSgov/mint-app/blob/79190855a4ab76afe7a2257c6c1054fb179ab2ba/pkg/models/model_helpers.go#L44-L51